package.json

```diff
@@ -12,16 +12,21 @@
     "coverage": "jest --coverage"
   },
   "dependencies": {
-    "bootstrap": "^5.0.1",
+    "@babel/runtime-corejs3": "^7.14.7",
+    "@types/jquery": "2.0.47",
```

Comment:
@DianaVargasOrellana, I did not see any place where you are using JQuery, furthermore, I think we should not use JQuery since all things that we need are already provided by knockout and Axios. So, if we are not using a library, we should remove it from our dependencies.

Reply:
I have deleted the unused dependencies